tools/manifest/manifest.py

```diff
@@ -221,20 +223,28 @@ def update(self, tree, parallel=True):
                     to_update.append(source_file)
 
         if to_update:
+            logger.debug("Computing manifest update for %s items" % len(to_update))
             changed = True
 
         if parallel and len(to_update) > 25 and cpu_count() > 1:
+            processes = cpu_count()
+            if max_cpu_count is not None:
+                processes = min(processes, max_cpu_count)
```

Comment:
Shouldn't `max_cpu_count` have some default on platforms where we think there's a bug?

Reply:
I was going to ask your thoughts on that once I marked this ready for review :P. 

*marks ready for review*.

Ok, so you think we should be opinionated here? There is unfortunately no actual set value for triggering the bug; it's based on the value of the Windows `MAXIMUM_WAIT_OBJECTS` enum. Now this is **defacto** 64 in current Windows (but there's an off-by-one bug in cpython still, so 63), but nothing actually stops it being less or more than this.

In blinkpy I intend to restrict this to 32, with the argument that that is still lots of multiprocessing for `wpt manifest` and is safely below 64. So I'd be happy to do similar here, but I wasn't sure if we wanted to given we haven't seen issues in Gecko or upstream WPT.

If you think we should hardcode a default, I'd be interested in knowing your opinion on whether `--max-cpu-count` is still a useful flag to have then. We could just drop it and say XY max cores is a feature/bug/restriction of `wpt manifest` on Windows.

Reply:
@jgraham Latest commit changes this to just hard-code a limit. Let me know if you prefer that over a command-line variable please :)